src/Orc.FileAssociation.Tests/FileAssociationsServiceFacts.cs

```diff
@@ -21,6 +24,20 @@ public async Task AssociateFilesWithApplicationAsyncTestAsync(ApplicationInfo ap
             Assert.AreEqual(expectedName, fileAssociationServiceMock.Name);
         }
 
+        [Test]
+        [Explicit]
+        public async Task OpenPropertiesWindowForExtensionAsyncTestAsync()
+        {
+            var path = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\\WildGums\\Temp\\";
```

Comment:
use Path.Combine instead of adding \\ manually between paths

Reply:
I think starting from Net.Core there is Path.Join, slightly better behavior in same cases (less possible mistakes)
https://docs.microsoft.com/ru-ru/dotnet/api/system.io.path.join?view=net-5.0